eap-xp4-openjdk17-image-stream.json

```diff
@@ -0,0 +1,123 @@
+{
```

Comment:
@luck3y Could you please remove this image stream? We will publish this image stream later when we have a better visibility of the availability of the JDK 17 images for EAP XP 4

Reply:
Yep, no problem.